### PR TITLE
Added links to sections in the header.

### PR DESCRIPTION
**Pull request by sudo-panda**
It solves the issue https://github.com/fossasia/gci16.fossasia.org/issues/142

Even tough the links don't work on the live gh-pages of my fork (https://baidya99.github.io/gci16.fossasia.org/), I have tested it on the original and it works there.

### CODE REVIEW

**Comment by hemantjadon** [comment]
Squash the commits please so that we can merge

File: index.html [hunk 1]
```diff
@@ -22,13 +22,22 @@
...
         </div>
+        <div class="sec1">
+            <a class="sec2" href="/#mentors">Mentors</a>
+            <a class="sec2" href="/#contributors">Contributers</a>
+            <a class="sec2" href="/#student-sec">Students</a>
+            <a class="sec2" href="/#blog">Blog</a>
+            <a class="sec2" href="/#mbdyn">Logos</a>
+            <a class="sec2" href="/#issues">Issues</a>
+            <a class="sec2" href="/#soc-med">Social</a>
+        </div>
```
Comment (shubham-padia) [inline index.html:40]: please remove the slashes preceding the `#` in the links, that is why the links are not working on your fork.

**Review by shubham-padia — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Baidya99) [inline index.html]: how do we squash commits online

**Comment by sudo-panda** [comment]
How to merge commits online.

**Pull request closed without merging**